TypeError: Cannot create property 'context' on string

Hi there

I have the following setup in place on a project running on node `7.2.1` and npm `3.10.10`.

```
{
  "scripts": {
    "lint:js": "semistandard",
    "lint:fix": "semistandard --fix",
    "lint:staged": "lint-staged",
  },
  "lint-staged": {
    "*.{js,jsx}": "lint:js"
  },
  "pre-commit": {
    "run": "lint:staged",
    "colors": true,
    "silent": true
  },
  "devDependencies": {
    "lint-staged": "^3.2.3",
    "pre-commit": "^1.2.2",
    "semistandard": "^9.2.1",
  }
}
```

All is okay here:

```
$ npm run lint:js

> example@0.0.0 lint:js /Users/mberridge/Sites/example
> semistandard | snazzy

semistandard: Semicolons For All! (https://github.com/Flet/semistandard)

/Users/mberridge/Sites/example/src/index.js
  9:1  error  Parsing error: Unexpected token

✖ 1 problem
```

But running `npm run lint:staged` returns an error wrapped around the output:

```
$ npm run lint:staged

> example@0.0.0 lint:staged /Users/mberridge/Sites/example
> lint-staged

 ❯ Running tasks for *.{js,jsx}
   ✖ lint:js
{ TypeError: Cannot create property 'context' on string '
🚫 lint:js found some errors. Please fix them and try committing again.

semistandard: Semicolons For All! (https://github.com/Flet/semistandard)

/Users/mberridge/Sites/example/src/index.js
  9:1  error  Parsing error: Unexpected token

✖ 1 problem

'
    at tasks.then.catch.err (/Users/mberridge/Sites/example/node_modules/lint-staged/node_modules/listr/index.js:74:17)
    at process._tickCallback (internal/process/next_tick.js:103:7) context: {} }
```

Not the end of the world, as it's still linting and working with `pre-commit` too. 
But have I done something wrong to cause the error?

Any help appreciated, thanks all!

--- a/src/index.js
+++ b/src/index.js
@@ -54,7 +54,7 @@ cosmiconfig('lint-staged', {
 
             if (tasks.length) {
                 new Listr(tasks, { concurrent }).run().catch((error) => {
-                    console.error(error)
+                    console.error(error.message)
                     process.exit(1)
                 })
             }

--- a/src/runScript.js
+++ b/src/runScript.js
@@ -17,13 +17,12 @@ module.exports = function runScript(commands, pathsToLint, packageJson, gitDir) 
                             resolve(`${ linter } passed!`)
                         })
                         .catch((err) => {
-                            reject(`
+                            reject(new Error(`
 🚫 ${ linter } found some errors. Please fix them and try committing again.
 
 ${ err.stderr }
 ${ err.stdout }
-`
-                            )
+`))
                         })
                 })
             } catch (err) {

